sqlmesh/core/notification_target.py

```diff
@@ -218,21 +253,51 @@ def send(self, notification_status: NotificationStatus, msg: str, **kwargs: t.An
 
 
 class BaseSlackNotificationTarget(BaseNotificationTarget):
-    def send(self, notification_status: NotificationStatus, msg: str, **kwargs: t.Any) -> None:
+    def send(
+        self,
+        notification_status: NotificationStatus,
+        msg: str,
+        audit_error: t.Optional[AuditError] = None,
+        exc: t.Optional[str] = None,
+        **kwargs: t.Any,
+    ) -> None:
+
         status_emoji = {
             NotificationStatus.PROGRESS: slack.SlackAlertIcon.START,
             NotificationStatus.SUCCESS: slack.SlackAlertIcon.SUCCESS,
             NotificationStatus.FAILURE: slack.SlackAlertIcon.FAILURE,
             NotificationStatus.WARNING: slack.SlackAlertIcon.WARNING,
             NotificationStatus.INFO: slack.SlackAlertIcon.INFO,
         }
+
         composed = slack.message().add_primary_blocks(
             slack.header_block(f"{status_emoji[notification_status]} SQLMesh Notification"),
-            slack.context_block(f"*Status:* {notification_status.value}"),
+            slack.context_block(f"*Status:* `{notification_status.value}`"),
+            slack.divider_block(),
+            slack.text_section_block(f"*Message*: {msg}"),
+        )
+
+        details = []
+        if audit_error:
+            details = [
+                slack.fields_section_block(
+                    f"*Audit*: `{audit_error.audit_name}`",
+                    f"*Model*: `{audit_error.model_name}`",
+                    f"*Count*: `{audit_error.count}`",
+                ),
+                slack.preformatted_rich_text_block(str(audit_error.sql(pretty=True))),
```

Comment:
Since `sql()` already returns a string
```suggestion
                slack.preformatted_rich_text_block(audit_error.sql(pretty=True)),
```